website/docs/basics/global-settings.md

```diff
@@ -0,0 +1,121 @@
+---
+id: global-settings
+title: Global Settings
+sidebar_label: Global Settings
+---
+
+[![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs)
+
+Ok, this is the part that we are incredibly proud of. Did you know that you can pass values from the JSON manifest or some other file directly to SCSS? Yes, I had the same facial expression when I found out. For this, to work, we used Webpack and its excellent features.
```

Comment:
```suggestion
Ok, this is the part that we are incredibly proud of. Did you know that you can pass values from the JSON manifest or some other file directly to SCSS? Yes, we had the same facial expression when we found out. For this, to work, we used Webpack and its excellent features.
```

Maybe we should we just stick to the we instead of I.